utils/makedbicons.sh

```diff
@@ -39,15 +39,15 @@ fi
 copy() {
     # The source file is:
     SRC="${SRCDIR}/svg/${1}.svg"
-    if [ ! -f "$SRC" ];then
+    if [[ ! -f $SRC ]]; then
         echo -e "${RED}Cannot find source icon for ${2} (${SRC})${NC}"
         return
     fi
 
     # Copy the source file to the destination, keeping
     # the source file's extension
     DST="$DSTDIR/${2}.svg"
-    cp $SRC $DST
+    cp -- "$SRC" "$DST"
```

Comment:
Again, why this change?

Reply:
The `--` is just like in the earlier comment, and the ``""`` around $SRC and $DST are to prevent the usual issues with accidental globbing and field splitting.